runtime-linux.md

```diff
@@ -1,11 +1,5 @@
 # Linux Runtime
 
-## File descriptors
-
-By default, only the `stdin`, `stdout` and `stderr` file descriptors are kept open for the application by the runtime.
-The runtime MAY pass additional file descriptors to the application to support features such as [socket activation](http://0pointer.de/blog/projects/socket-activated-containers.html).
-Some of the file descriptors MAY be redirected to `/dev/null` even though they are open.
```

Comment:
On Fri, Oct 14, 2016 at 08:29:41AM -0700, Aleksa Sarai wrote:
> I don't see why that question is relevant…

Because it lets me know if you need further changes on this point beyond those made by [0530bda → a013890][1].  It sounds like you do not.

> Does my logic behind that point make sense to you now?

You're argument for API-agnostic guarantees has made sense to me since [here][2].  My reluctance was based on my expectation that all APIs would provide this functionality, so spelling it out [felt redundant][1].  But part of writing a spec is spelling out all the expectations, so I'm fine with the approach taken by 0530bda → a013890.

[1]: https://github.com/opencontainers/runtime-spec/pull/513#discussion_r83372698
[2]: https://github.com/opencontainers/runtime-spec/pull/513#discussion_r83235690